projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -370,6 +370,11 @@ private function build_task( $task ) {
 			}
 		}
 
+		$extended_content = $this->load_extended_content( $task );
+		if ( $extended_content ) {
```

Comment:
It may be worth validating that we have a non-empty array here, though I don't think that should be blocking.
```suggestion
		if ( is_array( $extended_content ) && $extended_content !== array() ) {
```